docs/website/blog/2022-01-14-odo-v2.5.0.md

```diff
@@ -37,23 +38,21 @@ Now you can delete resoures that were deployed using `odo deploy` command using
 |`odo delete --all`| YES (deletes all resources created by odo) | YES|
 
 
-
 ### Add suppport for cpuLimit, cpuRequest and memoryRequest ([PR#5252](https://github.com/redhat-developer/odo/pull/5252) [@anandrkskd](https://github.com/anandrkskd))
-odo now supports specifing additional resource contstrains for `container` components in Devfile as introduced in Devfile v2.1.0.
 
-You can use following constrains:
+odo now supports specifying additional resource constrains for `container` components in Devfile as introduced in Devfile v2.1.0.
+
+You can use the following constraints:
 | Devfile container field | Kubernetes equivalent in Pod definition | Description |
 | - | - | -|
 |`memoryLimit`| `spec.containers[].resources.limits.memory` | Describes the maximum allowed memory for the container. |
 |`memoryRequest`| `spec.containers[].resources.requests.cpu`  | Describes the minimum memory that the container requires. |
 |`cpuLimit`| `spec.containers[].resources.limits.cpu` | Describes the maximum allowed CPU cores for the container  |
 |`cpuRequest`| `spec.containers[].resources.requests.cpu` | Describes the minimum number of CPU cores that the container requires.|
 
-You can lear more about resource mamagement in [Resource Management for Pods and Containers](https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/) section in [Kubernetes documentation](https://kubernetes.io/docs/)
-
-
+You can learn more about resource management in [Resource Management for Pods and Containers](https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/) section in [Kubernetes documentation](https://kubernetes.io/docs/)
 
-Example of a Devfile container with all resource constrains:
+Example of a Devfile container with all resource constraints:
```

Comment:
maybe `all available resource constraints`? no grammar error here, just curious

Reply:
you are right, it sounds better